experimental/uefi/loader/src/server.rs

```diff
@@ -28,6 +28,17 @@ use proto::{
     UnaryRequest, UnaryResponse,
 };
 
+fn serialize_request(request: UnaryRequest) -> Vec<u8> {
+    // The payload is the request's body prepended with the 8 byte session_id.
```

Comment:
What does session id correspond to?
Is it a grpc or https session, or it's an uefi specific thing?

Reply:
It's sent by the client to identify requests as coming from the same client. In the current implementation sessions are done over gRPC, though that technically does not matter for the loader <-> UEFI link.